packages/calypso-products/src/get-storage-addon-display-name.ts

```diff
@@ -0,0 +1,14 @@
+import { translate } from 'i18n-calypso';
+
+export function getStorageAddOnDisplayName( productName: string, quantity: number | null ) {
+	if ( quantity ) {
+		return translate( '%(productName)s %(quantity)s GB', {
```

Comment:
Can you leave a comment for the translators here (you can see other examples in the codebase). They may not know what `productName`, `quantity` etc. mean here. And mention that GB is gigabytes. Once that's done, you can string freeze it.

Reply:
Thanks, updated